frontend/src/Editor/QueryManager/DataSourceLister.jsx

```diff
@@ -34,6 +42,10 @@ function DataSourceLister({ dataSources, staticDataSources, changeDataSource, ha
           </div>
         );
       })}
+      <div className="query-datasource-card" style={computedStyles} onClick={dataSourceModalHandler}>
+        <AddIcon style={{ height: 25, width: 25, marginTop: '-3px' }} />
+        <p style={{ fontWeight: '500' }}> Add datasource</p>
```

Comment:
Reduce the `fontWeight` to `400` and the icon color to `grey` as it looks different from other sources. 
<img width="630" alt="image" src="https://user-images.githubusercontent.com/50441969/172758833-b88bf000-5f7c-4b5f-8a2d-ef786456ee50.png">